bft/testers/datafusion/runner.py

```diff
@@ -150,6 +150,9 @@ def run_sql_case(self, case: Case, mapping: SqlMapping) -> SqlCaseResult:
             elif case.result == "nan":
                 if math.isnan(result):
                     return SqlCaseResult.success()
+            elif case.result.type.startswith('fp') and case.result.value:
+                if math.isclose(result, case.result.value, rel_tol=1e-6):
```

Comment:
Is the tolerance something we want to eventually fix?  If so, should we leave a comment explaining why this is necessary (perhaps a pointer to an issue)?

Reply:
In general, python has a problem with float comparison: https://tutorpython.com/python-mathisclose/#The_problem_with_using_for_float_comparison

So this might be something that we need to keep.  Maybe I can put a link to that in a comment?  I can also do more investigation on the types of rounding done between different backends.

Datafusion also has a bug with floats.  I've filed an issue here: https://github.com/apache/arrow-datafusion/issues/9950